### ISSUE TITLE
piecewise linear gradient boosting tree

### ISSUE THREAD

**Issue by deadsoul44** [dmlc/xgboost#7714]
This can be very helpful.

LGBM: https://lightgbm.readthedocs.io/en/latest/Parameters.html#linear_tree
Paper: https://arxiv.org/pdf/1802.05640.pdf

**Comment by abhishekde95** [comment]
+1 on this feature request. Has there been any development along these fronts?

**Comment by trivialfis** [comment]
Not yet, but it's under the radar.
